.kitchen.docker.yml

```diff
@@ -24,6 +24,7 @@ platforms:
     provision_command:
       - /usr/bin/apt-get update
       - /usr/bin/apt-get install apt-transport-https net-tools -y
+      - mkdir /etc/cron.hourly
```

Comment:
Was the issue that the Docker Ubuntu image doesn't have cron installed, but the standard Ubuntu server install _does_?